README.md

```diff
@@ -1,6 +1,12 @@
 PaStA - Patch Stack Analysis
 ============================
 
+About
+------------
+
+PaStA is a tool for analysing Patch Stacks or mailboxes.
+In general, a patch stack (also known as patch set) is defined as a set of patches (commits) that are developed and maintained independently of the base project.
```

Comment:
remove the "defines". I don't know of an "undefined set of patches" :-)

Reply:
I think you misread the statement.
> A patch stack is defined as a set of patches...

Just means that I am defining what a patch stack is.